src/main/java/net/sydokiddo/auditory/mixin/blocks/BlockSoundsMixin.java

```diff
@@ -18,117 +18,117 @@
 // This is entirely data-driven and can be controlled by adding the block's ID to the respective tag through a datapack or mod.
 
 @Mixin(Block.class)
-abstract class BlockMixin extends BlockBehaviour {
-    public BlockMixin(Properties properties) {
+abstract class BlockSoundsMixin extends BlockBehaviour {
+    public BlockSoundsMixin(Properties properties) {
         super(properties);
     }
 
     @Inject(at = @At("HEAD"), method = "getSoundType", cancellable = true)
-    public void getSoundType(BlockState state, CallbackInfoReturnable<SoundType> info) {
+    private void auditory_alterSoundType(BlockState state, CallbackInfoReturnable<SoundType> info) {
 
         if (Auditory.getConfig().block_sounds.block_sounds) {
 
             if (state.is(AuditoryTags.BASALT_SOUNDS)) {
                 info.setReturnValue(SoundType.BASALT);
             }
 
-            if (state.is(AuditoryTags.CLAY_BRICK_SOUNDS)) {
+            else if (state.is(AuditoryTags.CLAY_BRICK_SOUNDS)) {
```

Comment:
`setReturnValue()` does not cause the handler method to return, it causes the target method to return after the handler does. So the previous code continued to run if checks, even after finding a match. Using an if-else-if chain avoids this.